test/functional/rpc_packages.py

```diff
@@ -25,6 +25,9 @@
 )
 
 
+MAX_PACKAGE_COUNT = 25
```

Comment:
nit: Might be good to stick this somewhere into the framework so it can be reused in other test, similar to the constants on top of `test_framework/messages.py`.

Reply:
The constant is specific to packages and not used anywhere else. I think I'd prefer keeping it in here until we need to access it in a different test?